pyerrors/obs.py

```diff
@@ -975,7 +975,10 @@ def _filter_zeroes(names, deltas, idl, eps=Obs.filter_eps):
             new_names.append(name)
             new_deltas[name] = np.array(nd)
             new_idl[name] = ni
-    return (new_names, new_deltas, new_idl)
+    if new_names:
```

Comment:
Isn't the code identical for both cases?

Reply:
Indeed, I did not insert the correct piece in this version...